packages/app-desktop/gui/ErrorBoundary.tsx

```diff
@@ -30,6 +33,28 @@ interface Props {
 	children: any;
 }
 
+interface BannerProps {
+	isVisible: boolean;
+}
+
+const SwitchToNewEditorBanner = (props: BannerProps) => {
+
+	const handleSwitchToNewEditor = async () => {
+		Setting.setValue('editor.legacyMarkdown', false);
+		const message = _('You are now using the latest version of the Markdown editor.');
+		await dialogs.alert(message);
+	};
+
+	return <BannerContent
+		acceptMessage={_('Switch to the new editor')}
+		onAccept={handleSwitchToNewEditor}
+		visible={props.isVisible}
+	>
+		{_('The legacy Markdown editor appears to have crashed due to an incompatibility with a plugin. We recommend using the new editor.')}
+		<br/>
```

Comment:
Why is such a `<br/>` hack necessary here?

Reply:
No answer here

Reply:
I thought having the option in the same level as the message was strange. I didn't realize that this is a style choice from the project.